### ISSUE TITLE
Rule Proposoal: ternary style

### ISSUE THREAD

**Issue by glennreyes** [eslint/eslint#6004]
My pesonally preferred multi-lined ternary style:
```js
someCondition
? true
: false;
```

**When does this rule warn? Please describe and show example code:**
When the question mark is placed like after the condition, e.g:
```js
someCondition ?
true :
false;
```

wich could be another ternary style!

Ternaries on one line shouldn't throw a warning, e.g:
```js
someCondition ? true : false;
```

**Is this rule preventing an error or is it stylistic?**
stylistic

**Why is this rule a candidate for inclusion instead of creating a custom rule?**
It keeps ternary styles clean and consistent

**Are you willing to create the rule yourself?**
I would love to if I had enough time for that

**Comment by platinumazure** [comment]
Have you tried looking at `operator-linebreak`? Does it not do what you want?

**Comment by glennreyes** [comment]
I think it covers it very well. Thanks for reference!

**Comment by platinumazure** [comment]
@glennreyes Awesome! Please feel free to close this issue if your question has been answered.
